content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Color considerations
+description: Color is an important aspect of creating accessible, inclusive experiences. 
+---
+
+## Overview
+
+This guidance covers considerations that can't automatically be integrated into Primer. It supplements [accessibility guidance given in the Color portion of our Foundations documentation](https://primer.style/design/foundations/color#accessibility).
+
+### Design tokens
+
+The majority of color contrast considerations should be handled by [Primer color design tokens](https://primer.style/design/foundations/color#how-to-use-color-for-product-ui). Ensure that solutions for the following considerations integrate design tokens as a method of creating accessible experiences.
+
+### Determining contrast
+
+[WebAIM's Contrast Checker tool](https://webaim.org/resources/contrastchecker/) is the recommended way to calculate color contrast ratios.
+
+### The APCA
+
+The [Advanced Accessible Perceptual Contrast Algorithm](https://www.myndex.com/APCA/) (<abbr>APCA</abbr>) is a proposal and is **not normative**. In terms of WCAG, and therefore, MAS-C compliance, this means that color contrast is valid **only** if it meets WCAG 2.1 criteria. 
+
+A color value that is APCA-compliant **may not** be WCAG compliant until APCA is integrated into WCAG 3.0. WCAG 3.0 is still a work in progress and currently does not have an anticipated release date.
+
+## Compliance 
+
+The following guidance is based on one or more **normative** Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) success criterion. This means that it is objective requirements with implications that affect MAS-C compliance efforts, as well as international laws regarding accessibility.
```

Comment:
> This means that it is objective requirements with implications that affect MAS-C compliance efforts, as well as international laws regarding accessibility.

Is it possible that there is a mistake in here? `This means that it is objective requirements` I don't 100% get what this means, so it is hard for me to propose an improvement.

Reply:
Does [this example](https://github.com/primer/design/pull/560/commits/c8120e43a00aaefccad7feee2632937db43176fa) help?

Reply:
```suggestion
The following guidance is based on one or more **normative** Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) success criterion. This means that the guidance is based on objective requirements with implications that affect MAS-C compliance efforts, as well as international laws regarding accessibility.
```

Could we add this and does it represent what you intended with this paragraph.

Reply:
Done!